# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#result definition
bool result_mode
al_msgs/JointCmd[] result_joint_pos
geometry_msgs/Pose result_cartesian_pos
